"panic: assignment to entry in nil map" when using custom metrics as of 0.5.0

## I'm submitting a ...

- [X] bug report
- [ ] feature request

## What is the current behavior?

When using custom metrics, a Go stack trace is generated:
```
github.com/iamseth/oracledb_exporter/collector.(*Exporter).checkIfMetricsChanged(0xc000290000)
    /go/src/oracledb_exporter/collector/collector.go:353 +0x3e5
<snip>
panic: assignment to entry in nil map
```

## If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem

Specfying a `CUSTOM_METRICS` environment variable that points to a file with any custom metrics inside of it will do the trick.

## What is the expected behavior?

Custom metrics behave as they did previously in `0.4.4`.

## What is the motivation / use case for changing the behavior?

Using custom metrics. :smile: 

## Please tell us about your environment:

Version: 0.5.0

## NOTES

This appears to be a result of commit d0fbef97ca9e242477019fec3dc81e3f13d83b9d . Previously the `hashMap` was [being initialized in main.go](https://github.com/iamseth/oracledb_exporter/blob/ed103f54d78fe36a26febf3d24dab72842ef6257/main.go#L591) but since the refactoring this initialization step isn't being performed anywhere in `collector.go`, resulting in an attempt to index into an unitialized map.

--- a/collector/collector.go
+++ b/collector/collector.go
@@ -76,7 +76,7 @@ type Metrics struct {
 
 var (
 	additionalMetrics Metrics
-	hashMap           map[int][]byte
+	hashMap           = make(map[int][]byte)
 	namespace         = "oracledb"
 	exporterName      = "exporter"
 )

